package.json

```diff
@@ -63,18 +60,19 @@
     "commitlint": "^17.0.2",
     "del-cli": "^5.0.0",
     "eslint": "^8.4.1",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.6.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-prettier": "^5.1.3",
     "jest": "^28.1.1",
     "mock-fs": "^5.2.0",
-    "prettier": "^2.0.5",
-    "react-native-builder-bob": "^0.20.0",
+    "prettier": "^3.2.5",
```

Comment:
There was some problem with pretteir 3. I would keep it at 2. I've encountered it recently but do not remember the details.